starter-code/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+    render() {
+    return (
+
```

Comment:
the ```<header> ``` code you put in index.js should go here. You'll need to wrap everything in a div because you can only return one thing in react